### ISSUE TITLE
Netflix sign-in issue

### ISSUE THREAD

**Issue by iNatNateNathan** [Alex313031/thorium#884]
**System Details**
 - OS: [e.g.  Ubuntu 22.04, Windows 7, 8.1, 10, 11]
 - Thorium Version [e.g.126.0.6478.246]

**Problem**
Unable to sign-in to Netflix account on Thorium Browser
Netflix message: Something went wrong
We are having technical difficulties and are actively working on a fix. Please try again in a few minutes.
Done all of the troubleshooting in terms of netflix account sign in issue.
Have check current Thorium Browser version (updated).

This happens after an hour watching, after that browser not loading while tried watching a video in youtube, then going back to Netflix, that happens...

**Did you read the FAQ?**
https://thorium.rocks/faq

**Additional Notes**

**Comment by Alex313031** [comment]
Can you try clearing all cache and cookies for Netflix.

You can do that by closing all tabs that have netflix open,
then going to 

`chrome://settings/content/all?searchSubpage=netflix`

and then pressing the delete button, to delete all cookies/data for the site.

Then re-open Netflix and sign in again. Might also check `chrome://components/` and make sure "Widevine Content Decryption Module" is up to date.

**Comment by iNatNateNathan** [comment]
@Alex313031

Thank you for your response, already done that, also restart the pc, and have check if there's an update of thorium, still having the same message: We are having technical difficulties and are actively working on a fix. Please try again in a few minutes.

**Comment by theridon** [comment]
I don't believe Netflix works on thorium due to netflix's bullshit drm policies

**Comment by iNatNateNathan** [comment]
Ty appreciate your response, sorry for late reply. I fixed the problem,
it's within netflix itself. I had to grand permision in signing-in the
account to thorium. All works well now, thank you again
